docs_user/modules/proc_adopting-the-compute-service.adoc

```diff
@@ -152,11 +137,11 @@ echo $PULL_OPENSTACK_CONFIGURATION_NOVAMANAGE_CELL_MAPPINGS
 oc rsh nova-cell0-conductor-0 nova-manage cell_v2 list_cells | grep -F '| cell1 |'
 ----
 +
-The expected changes to happen:
+The following changes are expected:
 
- ** cell1's `nova` DB and user name become `nova_cell1`.
- ** Default cell is renamed to `cell1` (in a multi-cell setup, it should become indexed as the last cell instead).
- ** RabbitMQ transport URL no longer uses `guest`.
+* The cell1 `nova` database and username become `nova_cell1`.
+* The default cell is renamed to `cell1`. In a multi-cell setup, it should become indexed as the last cell instead.
```

Comment:
@SeanMooney I’m not sure if we need to say “In a multi-cell setup, it should become indexed as the last cell instead.” This procedure is focused on a single cell configuration, so I suggest staying focused on that.

Reply:
that is debatable. 
the Bogdan is currently working on the support for multiple compute cells

we are generally trying to remove the single cell and multi-cell disticiton form all of our docs

on of the open questions we are currently resolving is if we can have a common adoption procedure or not.


all OpenStack deployments have at least 2 cells 

cell-0 which is reserved for instance that could not be scheduled to any host and some other internal nova usage and a compute cell typically called "cell-1" or sometimes "default"

when i refer to a comptue cell i mean a cell other then cell-0
or endgoal is to have one procedure for both cases since we will no longer be maintaining a difference in our docs for a cloud that only has 2 cells (cell-0 and either cell-1 or default) and a cloud that has more the two cells.

Reply:
@klgill for context this is the current work in flight to update the procedure to work with multiple compute cells https://github.com/openstack-k8s-operators/data-plane-adoption/pull/517

Reply:
I have discovered a regression in docs, which makes it disconnected from tests, see https://github.com/openstack-k8s-operators/data-plane-adoption/pull/490#issuecomment-2228606100

Reply:
regarding the multi-cell work in progress, I can make it rebasing on top of this, hopefully. Ideally though I agree this should be blocked on finishing the multi-cell guide in the PR that Sean has linked.

However, if this need to proceed now, feel free to just omit any confusing sections and statements related to multi-cell arch, as it was in flux by the time of writing, and still it is

Reply:
@bogdando This procedure is intended to be published in the adoption guide async after GA. It will likely be published before Feature Release 1, when the multi-cell feature is finished. 
I'm inclined to do as you suggested: Remove any statements related to multi-cell and then merge. Let me know if anything else should be done.

Reply:
+1, i think this PR contains a bunch of refinements that are unrelated to the multi-cell situation? So ideally we wouln't prevent this from landing altogether.

Reply:
FYI @SeanMooney @bogdando @jistr : I hid/removed all multi-cell references. Let me know if I missed anything, or if you have additional comments on this procedure in general. Thank you!